webapp/src/components/ProgramCalendar.tsx

```diff
@@ -53,7 +56,10 @@ const ProgramCalendar = ({
     allDay: false,
     start: new Date(),
```

Comment:
I'm thinking the defaults for start and end maybe should have a different value, perhaps `new Date(0)`. I know it's not in scope for your ticket but while you're here anyway, might as well!